src/main/java/christmas/model/Menu.java

```diff
@@ -0,0 +1,69 @@
+package christmas.model;
+
+import static christmas.config.CommonConfig.MENU_NOT_FOUND_ERROR;
+import static christmas.exception.IllegalArgumentExceptionThrower.throwException;
+import static christmas.model.MenuDivision.APPETIZER;
+import static christmas.model.MenuDivision.DESSERT;
+import static christmas.model.MenuDivision.DRINK;
+import static christmas.model.MenuDivision.MAIN;
+
+public enum Menu {
+    MUSHROOM_SOUP("양송이수프", 6000, APPETIZER),
+    TAPAS("타파스", 5500, APPETIZER),
+    CAESAR_SALAD("시저샐러드", 8000, APPETIZER),
+    T_BONE_STEAK("티본스테이크", 55000, MAIN),
```

Comment:
정말 사소하지만 하나의 `enum`이 여러 개의 하위 카테고리로 구분될 수 있다면 하위 카테고리마다 개행으로 구분해 준다면 추후에 필요한 값을 찾기 훨씬 쉬울거에요!
```suggestion
    CAESAR_SALAD("시저샐러드", 8000, APPETIZER),
    
    T_BONE_STEAK("티본스테이크", 55000, MAIN),
```